lib/platform/edgehooks/debugRequest/debugRequest.hooks.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @function
+ * @name DebugRequest
+ * @memberof edge
+ * @description
+ * The `DebugRequest` function is used to debug and display the details of an incoming request.
+ * It takes the request as a parameter and logs the request URL, method, and headers to the console.
+ * @param {FetchEvent} event - The incoming FetchEvent object.
+ * @returns {Promise<void>} A promise that resolves once the request details are logged.
+ */
+async function DebugRequest(event) {
+  const { request } = event;
+  const headers = new Headers(request.headers);
+
+  const requestData = {
+    url: request.url,
+    method: request.method,
+    headers: {},
+  };
+
+  headers.forEach((value, name) => {
+    requestData.headers[name] = value;
+  });
+
+  return event.console.log(requestData);
```

Comment:
if we keep the debug method in the preset handler, maybe we can log the information only when using the DEBUG flag ?

Reply:
removed hourly to avoid unnecessary computation/complications. we will be back in the future.